app/src/components/TermsOfUse.jsx

```diff
@@ -2,355 +2,359 @@ import React, { Component } from 'react';
 import CoverPrimary from './Shared/CoverPrimary';
 import Footer from './Shared/Footer';
 import { Link } from 'react-router';
+import textPages from '../../styles/components/c-text-pages.scss';
```

Comment:
for import SASS files we are using the nomenclature  _ModuleNameStyles_ . Please change it to be consistent.